app/javascript/components/download/DownloadPopup.vue

```diff
@@ -11,6 +11,14 @@
           @click="clickOption(option)"
           v-html="option.title"
         />
+        <a
+          v-else-if="option.isMap"
+          class="popup__link"
+          :href="option.url"
+          :download="option.title"
+          :title="option.title"
+          v-html="option.title"
+        />
```

Comment:
This works well!
Although I've noticed that the map link is always underlined, perhaps we should adjust the style a bit.

![Screen Shot 2020-09-29 at 16 16 31](https://user-images.githubusercontent.com/1244329/94578152-44925c00-026f-11eb-98dd-6f23b01630fa.png)

Reply:
@stacytalbot Noticed that anchor tags were always set to be underlined in `base.scss` - can I remove it for `popup__link`?